src/StoreKeeper/WooCommerce/B2C/Endpoints/Webhooks/InfoHandler.php

```diff
@@ -171,13 +171,18 @@ public static function getOrderSystemStatus(): array
 
         $orderSystemStatus['last_synchronized_date'] = TaskModel::getLatestSuccessfulSynchronizedDateForType(TaskHandler::ORDERS_EXPORT);
 
+        $orderStatuses = wc_get_order_statuses();
+        unset($orderStatuses['wc-cancelled']);
+
         $failedOrderIds = TaskModel::getFailedOrderIds();
         $failedOrdersQuery = new \WC_Order_Query([
             'post__in' => $failedOrderIds,
             'orderby' => 'ID',
             'order' => 'ASC',
+            'status', array_keys($orderStatuses),
```

Comment:
```suggestion
            'status' => array_keys($orderStatuses),
```

Isn't that the issue?

Reply:
I was wrong with this indeed but the issue is still valid.
![image](https://github.com/storekeeper-company/storekeeper-woocommerce-b2c/assets/18332309/3cb481a2-f282-4753-a3f5-0f60fd807e57)

Reply:
hmm can you remove the status filter line than? i'll approve after